lib/beaker/dsl/helpers.rb

```diff
@@ -1,19 +1,14 @@
 # -*- coding: utf-8 -*-
-require 'resolv'
-require 'inifile'
-require 'timeout'
-require 'beaker/dsl/outcomes'
-require 'beaker/options'
-require 'hocon'
-require 'hocon/config_error'
+[ 'facter', 'hiera', 'host', 'puppet', 'tk', 'web' ].each do |lib|
+      require "beaker/dsl/helpers/#{lib}_helpers"
+end
```

Comment:
Definitely personal preference, but I'd probably just do the 6 `require` calls directly (especially since the 6 module includes are literal just below).